audbackend/core/backend.py

```diff
@@ -232,8 +235,15 @@ def get_file(
     ) -> str:
         r"""Get file from backend.
 
-        The operation is silently skipped,
-        if a local file with the same checksum already exists.
+        If the folder of
+        ``dst_path`` does not exist,
+        it is created.
+
+        If ``dst_path`` exists
+        with a different checksum,
```

Comment:
This does mean we also replace an existing folder by a file, correct?
If yes, we should highlight this more as this might not be what a user expects.

Reply:
I don't think a file system allows to replace a folder with a file.

Reply:
I guess then we need to add that error to the "Raises" section.

Reply:
But to be on the safe side we should probably raise an error if the folder exists, which we are currently not doing.  I created https://github.com/audeering/audbackend/issues/121